common/src/main/java/com/wynntils/models/abilities/ShamanTotemModel.java

```diff
@@ -176,21 +176,22 @@ public void onTimerSpawn(AddEntityEvent e) {
     }
 
     @SubscribeEvent
-    public void onTotemRename(SetEntityDataEvent e) {
+    public void onTotemRename(EntityLabelChangedEvent e) {
         if (!Models.WorldState.onWorld()) return;
 
-        int entityId = e.getId();
-        Entity entity = getBufferedEntity(entityId);
+        Entity entity = e.getEntity();
         if (!(entity instanceof ArmorStand)) return;
 
-        String name = getNameFromMetadata(e.getPackedItems());
```

Comment:
Is `getNameFromMetadata` used anywhere else? Can we remove that too?

Reply:
It will be used for mob totem tracking, but at the moment, this should be it (just checking from e.getPackedItems())